relay-cardinality/src/redis/state.rs

```diff
@@ -0,0 +1,154 @@
+use relay_statsd::metric;
```

Comment:
Do you want to add module level docs as well?

Reply:
I can but the module is private (like all submodules of the `redis` module) and don't show up on the built rust docs, I think all the relevant info is already on the individual structs and that's the ones you'd see in the IDE.